mlir/include/mlir/Dialect/Rock/IR/Rock.h

```diff
@@ -55,6 +55,7 @@ ArrayAttr getIndexArrayAttr(Builder &b, ArrayRef<int64_t> values);
 } // end namespace rock
 } // end namespace mlir
 
+#include "mlir/Dialect/Rock/IR/RockTuningParamAttrInterface.h.inc"
```

Comment:
Don't use the `.inc` here? You already have it included in `RockTuningParamAttrInterface.h`